src/systems/ec/ec-component/ec-component-navigation/ec-component-navigation-menu/ec-component-navigation-menu.scss

```diff
@@ -1,334 +1,17 @@
-}
-
-.ecl-navigation-menu__hamburger--squeeze
-  .ecl-navigation-menu__hamburger-inner::after {
-  transition: bottom 0.075s 0.12s ease,
-    transform 0.075s cubic-bezier(0.55, 0.055, 0.675, 0.19);
-}
-
-.ecl-navigation-menu__hamburger--squeeze[aria-expanded='true']
-  .ecl-navigation-menu__hamburger-inner {
-  transform: rotate(45deg);
-  transition-delay: 0.12s;
-  transition-timing-function: cubic-bezier(0.215, 0.61, 0.355, 1);
-}
-
-.ecl-navigation-menu__hamburger--squeeze[aria-expanded='true']
-  .ecl-navigation-menu__hamburger-inner::before {
-  opacity: 0;
-  top: 0;
-  transition: top 0.075s ease, opacity 0.075s 0.12s ease;
-}
-
-.ecl-navigation-menu__hamburger--squeeze[aria-expanded='true']
-  .ecl-navigation-menu__hamburger-inner::after {
-  bottom: 0;
-  transform: rotate(-90deg);
-  transition: bottom 0.075s ease,
-    transform 0.075s 0.12s cubic-bezier(0.215, 0.61, 0.355, 1);
-}
-
-@include ecl-media-breakpoint-up(md) {
-  .ecl-navigation-menu {
-    background-color: map-get($ecl-colors, 'blue-25');
-  }
-
-  .ecl-navigation-menu__root {
-    background-color: transparent;
-    flex-direction: row;
-  }
-
-  .ecl-navigation-menu__root[aria-hidden='true'] {
-    display: flex;
-  }
-
-  .ecl-navigation-menu__root::before {
-    display: none;
-  }
-
-  .ecl-navigation-menu__toggle {
-    display: none;
-  }
-
-  .ecl-navigation-menu__group {
-    flex-wrap: wrap;
-    left: 0;
-    padding: map-get($ecl-spacing, 'xs') 0 map-get($ecl-spacing, 's');
-    position: absolute;
-    top: 100%;
-    width: 100%;
-  }
-
-  .ecl-navigation-menu__link {
-    border-bottom-width: 0;
-    font-size: map-get($ecl-font-size, 'xs');
-    margin: 0;
-    padding: map-get($ecl-spacing, 's');
-  }
-
-  .ecl-navigation-menu__group .ecl-navigation-menu__link {
-    padding: map-get($ecl-spacing, 'xs') map-get($ecl-spacing, 's')
-      map-get($ecl-spacing, 'xs') 0;
-  }
-
-  .ecl-navigation-menu__link[aria-expanded]::after {
-    position: relative;
-  }
-
-  .ecl-navigation-menu__link[aria-expanded='true'] {
-    background-color: map-get($ecl-colors, 'grey-5');
-    color: map-get($ecl-colors, 'blue-120');
-  }
-
-  .ecl-navigation-menu__link[aria-expanded='true']
-    ~ .ecl-navigation-menu__group {
-    align-items: baseline;
-    display: flex;
-    z-index: map-get($ecl-z-index, 'highlight');
-  }
-
-  .ecl-navigation-menu__item {
-    display: flex;
-  }
-
-  .ecl-navigation-menu__group .ecl-navigation-menu__item {
-    flex-basis: 25%;
-  }
-
-  .ecl-navigation-menu__item--active {
-    position: static;
-  }
-
-  .ecl-navigation-menu__item--active::before {
-    display: none;
-  }
-
-  .ecl-navigation-menu__item--active
-    > .ecl-navigation-menu__link[aria-expanded='true'] {
-    color: map-get($ecl-colors, 'blue-120');
-  }
-}
+@import '@ecl/generic-component-navigation-menu/generic-component-navigation-menu';
+
+@include ecl-navigation-menu(
+  $ecl-navigation-menu-bar-bg-mobile: map-get($ecl-colors, 'blue-75'),
```

Comment:
`$ecl-navigation-menu-bar-bg-mobile: map-get($ecl-colors, 'blue-75'),` vs `$bar-bg-mobile: map-get($ecl-colors, 'blue-75'),`?

Reply:
to be discussed...